hwy/ops/rvv-inl.h

```diff
@@ -2054,6 +2121,20 @@ HWY_API vint32m4_t DemoteTo(Simd<int32_t, N, 2> d, const vfloat64m8_t v) {
   return __riscv_vfncvt_rtz_x_f_w_i32m4(v, Lanes(d));
 }
 
+// SEW is for the source so we can use _DEMOTE_TO_SHR_16.
```

Comment:
The reason why I had _DEMOTE_TO_SHR_16 in the comment above is that the comment was copied from a comment that had just _DEMOTE instead of _DEMOTE_VIRT.

Reply:
Thanks for the heads-up, I am updating the other comments as well.

_DEMOTE_VIRT was added after those comments were written. The comment refers to the choice of SEW vs SEWD (double) or SEWH (half) arguments.